addons/mail/static/src/components/message_list/message_list.xml

```diff
@@ -97,9 +97,11 @@
             </div>
         </t>
         <t t-elif="!threadView.threadCache.isAllHistoryLoaded">
-            <a class="o_MessageList_item o_MessageList_loadMore" href="#" t-on-click="_onClickLoadMore" t-ref="loadMore">
-                Load more
-            </a>
+            <t t-if="!threadView.thread.isTemporary">
```

Comment:
add the condition on the `elif` just above, no need for a separate `t-if` if there are no further branching

Reply:
Also need to update test "base rendering when chatter has no record" to include the check for the absence of this button.

Reply:
> add the condition on the elif just above, no need for a separate t-if if there are no further branching

That's what I did in the first place. But it wasn't working. I forgot Owl use `and` for `&&`.

Reply:
done !